spritz/spritz-android/src/main/java/com/novoda/spritz/Spritz.java

```diff
@@ -1,249 +1,86 @@
 package com.novoda.spritz;
 
-import android.animation.Animator;
-import android.animation.AnimatorSet;
-import android.animation.TimeInterpolator;
-import android.animation.ValueAnimator;
 import android.support.v4.view.ViewPager;
-import android.view.animation.LinearInterpolator;
 
 import com.airbnb.lottie.LottieAnimationView;
 
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
 import java.util.List;
-import java.util.concurrent.TimeUnit;
 
 public class Spritz {
 
-    private final LottieAnimationView lottieAnimationView;
     private final List<SpritzStepWithOffset> spritzStepsWithOffset;
-    private final long totalAnimationDuration;
-    private final long defaultSwipeAnimationDuration;
-    private final TimeInterpolator defaultSwipeForwardInterpolator;
-    private final TimeInterpolator defaultSwipeBackwardsInterpolator;
+    private final SpritzCalculator spritzCalculator;
```

Comment:
i'm having a bit of an issue with having everything named `SpritzXYZ` (having `NSBlaBlaBla` flashbacks) but i can't come up with much better names atm (i know there's the risk of collision for at least one or two). just wondering if you considered this (or something similar): `PagerAnimation`, `PagerAnimator`, etc.

Reply:
Yeah, I had the same feeling about this. Maybe I'll rename to `Animation` and `AnimationRunner`?

Reply:
`SpritzAnimation` ➡️  `Animation` and `SpritzAnimator` ➡️  `AnimationRunner`? sounds good

Reply:
Renamed in ba041d3f0f6cc7b70501ded73a5fd42b484fc726:
- `SpritzAnimation` to `Animation`
- `SpritzAnimator` to `AnimationRunner`
- `SpritzCalculator` to `ProgressCalculator`